cpp/daal/src/algorithms/k_nearest_neighbors/oneapi/bf_knn_classification_predict_kernel_ucapi_impl.i

```diff
@@ -74,8 +74,8 @@ private:
 };
 
 template <typename algorithmFpType>
-services::Status KNNClassificationPredictKernelUCAPI<algorithmFpType>::compute(const NumericTable * x, const classifier::Model * m, NumericTable * y,
-                                                                               const daal::algorithms::Parameter * par)
+services::Status KNNClassificationPredictKernelUCAPI<algorithmFpType>::compute(const NumericTable * x, const classifier::Model * m, NumericTable * y, NumericTable * outIndices,
```

Comment:
This line is too long and the end of line can't be seen on github

Reply:
I've fixed it. Please check